packages/bootstrap-vue-next/src/components/BFormGroup/BFormGroup.vue

```diff
@@ -58,20 +58,20 @@ export default defineComponent({
     labelColsXl: {type: [Boolean, String, Number], default: undefined},
     labelFor: {type: String, default: undefined},
     labelSize: {type: String, default: undefined},
-    labelSrOnly: {type: [Boolean, String] as PropType<Booleanish>, default: false},
-    state: {type: [Boolean, String] as PropType<Booleanish | null>, default: null},
-    tooltip: {type: [Boolean, String] as PropType<Booleanish>, default: false},
+    labelSrOnly: {type: [Boolean, String] as PropType<boolean>, default: false},
+    state: {type: [Boolean, String] as PropType<boolean | null>, default: null},
+    tooltip: {type: [Boolean, String] as PropType<boolean>, default: false},
```

Comment:
```suggestion
    labelSrOnly: {type: Boolean, default: false},
    state: {type: Boolean as PropType<boolean | null>, default: null},
    tooltip: {type: Boolean, default: false},
```

`state: {type: Boolean as PropType<boolean | null>, default: null},` probably has some alternative syntax that doesn't use `PropType`